### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by er-minio** [iloveitaly/homebrew-tap#2]
I'm getting (amongst other) this error when trying to install: 

`Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist.
`
Is this something on my end?

**Comment by iloveitaly** [comment]
@er-minio for which package and on which platform?

**Comment by er-minio** [comment]
Hi @iloveitaly apologies for the poorly written request!

I'm trying to install hyper focus on MacOs X 13.2.1 (22D68). Xcode installed.

**Comment by iloveitaly** [comment]
Can you try again with verbose logging enabled and paste/screenshot the output here?

**Comment by er-minio** [comment]
Here it is.

`er-minio@Mac-Studio ~ % brew install -v iloveitaly/tap/hyper-focus        

Running `brew update --auto-update`...
==> Auto-updated Homebrew!
Updated 2 taps (homebrew/core and homebrew/cask).
==> New Formulae
aarch64-elf-gdb        dexter                 flavours               ksops                  notify                 trurl                  wazero
blocky                 enchive                hz                     libansilove            renovate               typst
==> New Casks
alipay-key-tool   beardie           cursor            dehelper          hummingbird       mullvad-browser   orcaslicer        usmart-trade      vbrokers

==> Fetching iloveitaly/tap/hyper-focus
==> Downloading https://github.com/iloveitaly/hyper-focus/archive/refs/tags/v0.1.9.tar.gz
Already downloaded: /Users/er-minio/Library/Caches/Homebrew/downloads/709af88ee73b222b69e3ada32921e8e549b1d9b084037b1c5e93c04a37f5eec7--hyper-focus-0.1.9.tar.gz
==> Verifying checksum for '709af88ee73b222b69e3ada32921e8e549b1d9b084037b1c5e93c04a37f5eec7--hyper-focus-0.1.9.tar.gz'
==> Installing hyper-focus from iloveitaly/tap
/usr/bin/env tar --extract --no-same-owner --file /Users/er-minio/Library/Caches/Homebrew/downloads/709af88ee73b222b69e3ada32921e8e549b1d9b084037b1c5e93c04a37f5eec7--hyper-focus-0.1.9.tar.gz --directory /private/tmp/d20230405-6098-14t58h1
/usr/bin/env cp -pR /private/tmp/d20230405-6098-14t58h1/hyper-focus-0.1.9/. /private/tmp/hyper-focus-20230405-6098-uug2gq/hyper-focus-0.1.9
==> swift build --disable-sandbox -c release
error: terminated(1): /usr/bin/xcrun --sdk macosx --show-sdk-platform-path output:
    xcrun: error: unable to lookup item 'PlatformPath' from command line tools installation
    xcrun: error: unable to lookup item 'PlatformPath' in SDK '/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk'
    

==> Formula
Tap: iloveitaly/tap
Path: /opt/homebrew/Library/Taps/iloveitaly/homebrew-tap/Formula/hyper-focus.rb
==> Configuration
HOMEBREW_VERSION: 4.0.11
ORIGIN: https://github.com/Homebrew/brew
HEAD: 931327df1f30a3a5b1aa07657c0d1070e61988f9
Last commit: 5 days ago
Core tap JSON: 05 Apr 09:01 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.87.0 => /usr/bin/curl
macOS: 13.3-arm64
CLT: 14.3.0.0.1.1679647830
Xcode: 14.2
Rosetta 2: false
==> ENV
HOMEBREW_CC: clang
HOMEBREW_CXX: clang++
MAKEFLAGS: -j10
CMAKE_PREFIX_PATH: /opt/homebrew
CMAKE_INCLUDE_PATH: /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Headers
CMAKE_LIBRARY_PATH: /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/System/Library/Frameworks/OpenGL.framework/Versions/Current/Libraries
PKG_CONFIG_LIBDIR: /usr/lib/pkgconfig:/opt/homebrew/Library/Homebrew/os/mac/pkgconfig/13
HOMEBREW_GIT: git
HOMEBREW_SDKROOT: /Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk
PATH: /opt/homebrew/Library/Homebrew/shims/mac/super:/usr/bin:/bin:/usr/sbin:/sbin

Error: iloveitaly/tap/hyper-focus 0.1.9 did not build
Logs:
     /Users/er-minio/Library/Logs/Homebrew/hyper-focus/01.swift
     /Users/er-minio/Library/Logs/Homebrew/hyper-focus/00.options.out
If reporting this issue please do so at (not Homebrew/brew or Homebrew/homebrew-core):
  https://github.com/iloveitaly/homebrew-tap/issues
`

**Comment by er-minio** [comment]
I fixed the PlatformPath as suggested here: https://stackoverflow.com/questions/43418533/running-swift-build-in-terminal-leading-to-platform-path-errors

Result:

`er-minio@Mac-Studio ~ % brew install -v iloveitaly/tap/hyper-focus                          

==> Fetching iloveitaly/tap/hyper-focus
==> Downloading https://github.com/iloveitaly/hyper-focus/archive/refs/tags/v0.1.9.tar.gz
Already downloaded: /Users/er-minio/Library/Caches/Homebrew/downloads/709af88ee73b222b69e3ada32921e8e549b1d9b084037b1c5e93c04a37f5eec7--hyper-focus-0.1.9.tar.gz
==> Verifying checksum for '709af88ee73b222b69e3ada32921e8e549b1d9b084037b1c5e93c04a37f5eec7--hyper-focus-0.1.9.tar.gz'
==> Installing hyper-focus from iloveitaly/tap
/usr/bin/env tar --extract --no-same-owner --file /Users/er-minio/Library/Caches/Homebrew/downloads/709af88ee73b222b69e3ada32921e8e549b1d9b084037b1c5e93c04a37f5eec7--hyper-focus-0.1.9.tar.gz --directory /private/tmp/d20230405-7083-1uqv7ib
/usr/bin/env cp -pR /private/tmp/d20230405-7083-1uqv7ib/hyper-focus-0.1.9/. /private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9
==> swift build --disable-sandbox -c release
warning: /Users/er-minio/Library/org.swift.swiftpm/configuration is not accessible or not writable, disabling user-level configuration features.
warning: /Users/er-minio/Library/org.swift.swiftpm/security is not accessible or not writable, disabling user-level security features.
warning: /Users/er-minio/Library/Caches/org.swift.swiftpm is not accessible or not writable, disabling user-level cache features.
warning: 'hyper-focus-0.1.9': failed loading cached manifest for 'hyper-focus-0.1.9': Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000015b2700 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: 'hyper-focus-0.1.9': <unknown>:0: remark: did not find a prebuilt standard library for target 'arm64-apple-macos' compatible with this Swift compiler; building it may take a few minutes, but it should only happen once for this combination of compiler and target
<unknown>:0: note: cached module is out of date: '/var/folders/3d/096y9dg16gl1nssqz234ydwc0000gn/C/clang/ModuleCache/Swift-47B1HXUCQMPI.swiftmodule'
<unknown>:0: note: dependency is out of date: '/Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/usr/lib/swift/Swift.swiftmodule/arm64e-apple-macos.swiftinterface'
warning: 'hyper-focus-0.1.9': failed storing manifest for 'hyper-focus-0.1.9' in cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000015bad90 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: 'hyper-focus-0.1.9': failed closing cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000015b8960 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed loading cached manifest for 'hyper-focus-0.1.9': Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000015b52f0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed storing manifest for 'hyper-focus-0.1.9' in cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000015b34b0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed closing cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000015b0ff0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
Fetching https://github.com/envoy/Embassy.git
Fetching https://github.com/apple/swift-argument-parser
Fetching https://github.com/envoy/Ambassador
[1/557] Fetching ambassador
[169/2386] Fetching ambassador, embassy
[1875/13135] Fetching ambassador, embassy, swift-argument-parser
Fetched https://github.com/envoy/Embassy.git (2.38s)
Fetched https://github.com/apple/swift-argument-parser (2.38s)
Fetched https://github.com/envoy/Ambassador (2.38s)
Computing version for https://github.com/apple/swift-argument-parser
warning: failed loading cached manifest for 'swift-argument-parser': Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001488810 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed storing manifest for 'swift-argument-parser' in cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001488ea0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed closing cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001488f60 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
Computed https://github.com/apple/swift-argument-parser at 1.2.2 (0.54s)
Computing version for https://github.com/envoy/Ambassador
warning: failed loading cached manifest for 'ambassador': Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000014884b0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed storing manifest for 'ambassador' in cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000014a5440 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed closing cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000014a5380 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
Computed https://github.com/envoy/Ambassador at 4.0.5 (1.83s)
Computing version for https://github.com/envoy/Embassy.git
warning: failed loading cached manifest for 'embassy': Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001494540 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed storing manifest for 'embassy' in cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x60000148b0f0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed closing cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x60000148b060 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
Computed https://github.com/envoy/Embassy.git at 4.1.6 (1.77s)
warning: failed loading cached manifest for 'swift-argument-parser': Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x60000148c720 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed storing manifest for 'swift-argument-parser' in cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001489c80 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed closing cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001489c50 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed loading cached manifest for 'ambassador': Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000014941e0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed storing manifest for 'ambassador' in cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000014880f0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed closing cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001489c50 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed loading cached manifest for 'embassy': Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001494780 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed storing manifest for 'embassy' in cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x60000148afa0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: failed closing cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001488270 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
Creating working copy for https://github.com/apple/swift-argument-parser
Working copy of https://github.com/apple/swift-argument-parser resolved at 1.2.2
Creating working copy for https://github.com/envoy/Embassy.git
Working copy of https://github.com/envoy/Embassy.git resolved at 4.1.6
Creating working copy for https://github.com/envoy/Ambassador
Working copy of https://github.com/envoy/Ambassador resolved at 4.0.5
warning: 'swift-argument-parser': failed loading cached manifest for 'swift-argument-parser': Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000015422b0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: 'ambassador': failed loading cached manifest for 'ambassador': Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001542610 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: 'ambassador': failed storing manifest for 'ambassador' in cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001499410 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: 'ambassador': failed closing cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x60000149a1c0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: 'swift-argument-parser': failed storing manifest for 'swift-argument-parser' in cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001499440 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: 'swift-argument-parser': failed closing cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x60000149a4c0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: 'embassy': failed loading cached manifest for 'embassy': Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x60000149a520 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: 'embassy': failed storing manifest for 'embassy' in cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x600001492370 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
warning: 'embassy': failed closing cache: Error Domain=NSCocoaErrorDomain Code=4 "The file “manifests” doesn’t exist." UserInfo={NSFilePath=/Users/er-minio/Library/Caches/org.swift.swiftpm/manifests, NSUnderlyingError=0x6000014921c0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
Compiling plugin GenerateManual...
Building for production...
remark: Incremental compilation has been disabled: it is not compatible with whole module optimizationremark: Incremental compilation has been disabled: it is not compatible with whole module optimization[1/3] Compiling ArgumentParserToolInfo ToolInfo.swift
<unknown>:0: remark: did not find a prebuilt standard library for target 'arm64-apple-macos' compatible with this Swift compiler; building it may take a few minutes, but it should only happen once for this combination of compiler and target
remark: Incremental compilation has been disabled: it is not compatible with whole module optimization[2/4] Compiling Embassy Atomic.swift
remark: Incremental compilation has been disabled: it is not compatible with whole module optimization[3/5] Compiling Ambassador DataReader.swift
[4/5] Compiling ArgumentParser BashCompletionsGenerator.swift
remark: Incremental compilation has been disabled: it is not compatible with whole module optimization[5/6] Compiling hyper_focus Google Chrome.swift
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/action_handler.swift:13:9: warning: result of call to 'browserAction' is unused
        browserAction(data)
        ^            ~~~~~~
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/action_handler.swift:70:67: warning: string interpolation produces a debug description for an optional value; did you mean to make this explicit?
            error("invalid url (\(supersetUrlString)), skipping \(optionalSupersetUrl) \(optionalSubsetUrl)")
                                                                  ^~~~~~~~~~~~~~~~~~~
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/action_handler.swift:70:67: note: use 'String(describing:)' to silence this warning
            error("invalid url (\(supersetUrlString)), skipping \(optionalSupersetUrl) \(optionalSubsetUrl)")
                                                                  ^~~~~~~~~~~~~~~~~~~
                                                                  String(describing: )
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/action_handler.swift:70:67: note: provide a default value to avoid this warning
            error("invalid url (\(supersetUrlString)), skipping \(optionalSupersetUrl) \(optionalSubsetUrl)")
                                                                  ^~~~~~~~~~~~~~~~~~~
                                                                                      ?? <#default value#>
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/action_handler.swift:70:90: warning: string interpolation produces a debug description for an optional value; did you mean to make this explicit?
            error("invalid url (\(supersetUrlString)), skipping \(optionalSupersetUrl) \(optionalSubsetUrl)")
                                                                                         ^~~~~~~~~~~~~~~~~
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/action_handler.swift:70:90: note: use 'String(describing:)' to silence this warning
            error("invalid url (\(supersetUrlString)), skipping \(optionalSupersetUrl) \(optionalSubsetUrl)")
                                                                                         ^~~~~~~~~~~~~~~~~
                                                                                         String(describing:  )
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/action_handler.swift:70:90: note: provide a default value to avoid this warning
            error("invalid url (\(supersetUrlString)), skipping \(optionalSupersetUrl) \(optionalSubsetUrl)")
                                                                                         ^~~~~~~~~~~~~~~~~
                                                                                                           ?? <#default value#>
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/schedule_manager.swift:103:41: warning: string interpolation produces a debug description for an optional value; did you mean to make this explicit?
            log("changing schedule to \(schedule ?? nil)")
                                        ^~~~~~~~~~~~~~~
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/schedule_manager.swift:103:50: note: use 'String(describing:)' to silence this warning
            log("changing schedule to \(schedule ?? nil)")
                                        ~~~~~~~~~^~~~~~
                                        String(describing:  )
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/schedule_manager.swift:103:50: note: provide a default value to avoid this warning
            log("changing schedule to \(schedule ?? nil)")
                                        ~~~~~~~~~^~~~~~
                                                        ?? <#default value#>
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:48:33: warning: expression implicitly coerced from 'String?' to 'Any'
                        "name": self.scheduleManager.plannedSchedule?.name,
                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:48:71: note: provide a default value to avoid this warning
                        "name": self.scheduleManager.plannedSchedule?.name,
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~
                                                                           ?? <#default value#>
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:48:71: note: force-unwrap the value to avoid this warning
                        "name": self.scheduleManager.plannedSchedule?.name,
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~
                                                                          !
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:48:71: note: explicitly cast to 'Any' with 'as Any' to silence this warning
                        "name": self.scheduleManager.plannedSchedule?.name,
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~
                                                                           as Any
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:49:34: warning: expression implicitly coerced from 'Int?' to 'Any'
                        "until": scheduledUntil,
                                 ^~~~~~~~~~~~~~
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:49:34: note: provide a default value to avoid this warning
                        "until": scheduledUntil,
                                 ^~~~~~~~~~~~~~
                                                ?? <#default value#>
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:49:34: note: force-unwrap the value to avoid this warning
                        "until": scheduledUntil,
                                 ^~~~~~~~~~~~~~
                                               !
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:49:34: note: explicitly cast to 'Any' with 'as Any' to silence this warning
                        "until": scheduledUntil,
                                 ^~~~~~~~~~~~~~
                                                as Any
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:52:34: warning: expression implicitly coerced from 'TimeInterval?' (aka 'Optional<Double>') to 'Any'
                        "until": self.scheduleManager.endOverride?.timeIntervalSince1970,
                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:52:68: note: provide a default value to avoid this warning
                        "until": self.scheduleManager.endOverride?.timeIntervalSince1970,
                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
                                                                                         ?? <#default value#>
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:52:68: note: force-unwrap the value to avoid this warning
                        "until": self.scheduleManager.endOverride?.timeIntervalSince1970,
                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
                                                                                        !
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:52:68: note: explicitly cast to 'Any' with 'as Any' to silence this warning
                        "until": self.scheduleManager.endOverride?.timeIntervalSince1970,
                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
                                                                                         as Any
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:53:33: warning: expression implicitly coerced from 'String?' to 'Any'
                        "name": self.scheduleManager.overrideSchedule?.name,
                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:53:72: note: provide a default value to avoid this warning
                        "name": self.scheduleManager.overrideSchedule?.name,
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~
                                                                            ?? <#default value#>
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:53:72: note: force-unwrap the value to avoid this warning
                        "name": self.scheduleManager.overrideSchedule?.name,
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~
                                                                           !
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:53:72: note: explicitly cast to 'Any' with 'as Any' to silence this warning
                        "name": self.scheduleManager.overrideSchedule?.name,
                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~
                                                                            as Any
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:56:34: warning: expression implicitly coerced from 'TimeInterval?' (aka 'Optional<Double>') to 'Any'
                        "until": self.scheduleManager.endPause?.timeIntervalSince1970,
                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:56:65: note: provide a default value to avoid this warning
                        "until": self.scheduleManager.endPause?.timeIntervalSince1970,
                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
                                                                                      ?? <#default value#>
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:56:65: note: force-unwrap the value to avoid this warning
                        "until": self.scheduleManager.endPause?.timeIntervalSince1970,
                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
                                                                                     !
/private/tmp/hyper-focus-20230405-7083-bugwms/hyper-focus-0.1.9/Sources/hyper-focus/server.swift:56:65: note: explicitly cast to 'Any' with 'as Any' to silence this warning
                        "until": self.scheduleManager.endPause?.timeIntervalSince1970,
                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
                                                                                      as Any
[5/6] Linking hyper-focus
Build complete! (54.68s)
==> Cleaning
==> Finishing up
ln -s ../Cellar/hyper-focus/0.1.9/bin/hyper-focus hyper-focus
==> Caveats
To restart iloveitaly/tap/hyper-focus after an upgrade:
  brew services restart iloveitaly/tap/hyper-focus
Or, if you don't want/need a background service you can just run:
  /opt/homebrew/opt/hyper-focus/bin/hyper-focus --configuration /Users/er-minio/.config/focus/config.json
==> Summary
🍺  /opt/homebrew/Cellar/hyper-focus/0.1.9: 8 files, 2.3MB, built in 1 minute 1 second
==> Running `brew cleanup hyper-focus`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
er-minio@Mac-Studio ~ % 
`

**Comment by iloveitaly** [comment]
Nice, looks like it's working! Closing this out.
